RecoEgamma/ElectronIdentification/python/Identification/heepElectronID_HEEPV80_cff.py

```diff
@@ -0,0 +1,100 @@
+import FWCore.ParameterSet.Config as cms
+from PhysicsTools.SelectorUtils.centralIDRegistry import central_id_registry
+
+# Common functions and classes for ID definition are imported here:
+from RecoEgamma.ElectronIdentification.Identification.heepElectronID_tools import HEEP_WorkingPoint_V1,configureHEEPElectronID_V80
+
+#
+# The HEEP ID cuts V8.0 below are high energy optimised cuts
+# The cut values are taken from the twiki:
+#       https://twiki.cern.ch/twiki/bin/view/CMS/HEEPElectronIdentificationRun2
+# HEEP ID V8.0 is really V6.0 with a new name, the changes all happend to the object
+# I bumped the name to make it was clear its for 9X
+
+# The cut values for the  Barrel and Endcap
+idName = "heepElectronID-HEEPV80"
+WP_HEEP80_EB = HEEP_WorkingPoint_V1(
```

Comment:
is _V1 really necessary?
 [there is a lot still possible between 8.0 and 8.9 with the more major version number semantics]

Reply:
HEEP_WorkingPoint_V1 is an already existing function which is used by all the HEEP IDs so far. This versioning is more for internal bookkeeping of functions and is not supposed to be exposed to the user. If we add or remove a cut, we will would then define a V2 function.